lib/core/src/irods_configuration_keywords.cpp

```diff
@@ -178,10 +188,17 @@ namespace irods
     const char* const KW_CFG_ACCESS_TIME_BATCH_SIZE{"batch_size"};
     const char* const KW_CFG_ACCESS_TIME_RESOLUTION_IN_SECONDS{"resolution_in_seconds"};
 
-    // TLS server configurations
-    const char* const KW_CFG_TLS_CONFIGURATION{"tls"};
+    // TLS server configurations (inbound traffic)
+    const char* const KW_CFG_TLS_SERVER{"tls_server"};
     const char* const KW_CFG_TLS_CERTIFICATE_CHAIN_FILE{"certificate_chain_file"};
     const char* const KW_CFG_TLS_CERTIFICATE_KEY_FILE{"certificate_key_file"};
     const char* const KW_CFG_TLS_DH_PARAMS_FILE{"dh_params_file"};
+    // TLS server configurations (outbound traffic)
```

Comment:
should this be 'client'?

```suggestion
    // TLS client configurations (outbound traffic)
```

Reply:
The use of the word "server" indicates these properties are server-side only, hence the outbound/inbound qualification. I'll see if there's better wording which communicates that.

Reply:
When I first read 'TLS server configurations (outbound traffic)', it seems it could have meant 'outbound but in response to a client request'...

I don't think what you've got is wrong, maybe we just need another line of comment/clarifying words.

Reply:
Sure. Will adjust so that it's clear.

Reply:
Updated wording.